### PR TITLE
Add Troubleshooting Section to NFS persistent storage doc

### PR DESCRIPTION
**Pull request by screeley44**
@adellape @erinboyd @bparees 
Added additional configuration and troubleshooting section to the nfs persistent storage doc based on BZ (https://bugzilla.redhat.com/show_bug.cgi?id=1282733)

### CODE REVIEW

**Comment by erinboyd** [comment]
LGTM

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 1]
```diff
@@ -188,3 +188,24 @@ following:
...
+
+== Possible Additional Configurations and Troubleshooting
+Depending on what version of NFS is being used and how it is configured, there may be additional
+configuration steps needed for proper export and security mapping.  Below are some links and topics that
+may apply:
+
+`*NFSv4 mount incorrectly shows all files with ownership of nobody:nobody*`
+
+- Could be attributed to the ID mapping settings (/etc/idmapd.conf) on your NFS client and server, as NFSv4 utilizes ID mapping to ensure permissions are set properly on exported shares, if the domains of the client and server do not match then the permissions are mapped to nobody:nobody
+- For link:http://file.rdu.redhat.com/~maszulik/NFSv4%20mount%20incorrectly%20shows%20all%20files%20with%20ownership%20as%20nobody:nobody%20-%20Red%20Hat%20Customer%20Portal.html[more information] on how to diagnose and resolve this issue
```
Comment (bparees) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: for official docs we need to host/reference that article somewhere more official than @soltysh's personal directory :)

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 2]
```diff
@@ -188,3 +188,24 @@ following:
 pods. Either set the export to be owned by the container's primary UID, or give
 your pod group based access using `*SuppplementalGroups*`. See
 link:pod_security_context.html[Volume Security] for more information.
+
+
+[[nfs-additional-config-and-troubleshooting]]
+
+== Possible Additional Configurations and Troubleshooting
```
Comment (bparees) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: i'd remove "possible" from this text.

and make Configurations singular.

File: install_config/persistent_storage/persistent_storage_nfs.adoc (hunk 1 above)
Comment (screeley44) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: @bparees @adellape  - was wondering about that, any idea on where this article should be hosted to make it more offiical?

File: install_config/persistent_storage/persistent_storage_nfs.adoc (hunk 2 above)
Comment (screeley44) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: @bparees - done

**Comment by erinboyd** [comment]
@adellape is there a references section to the overall doc that we can move @soltysh stuff to?

**Comment by erinboyd** [comment]
How about linking to here:
https://access.redhat.com/solutions/33455

**Comment by screeley44** [comment]
@erinboyd :  yeah, that is what I was looking for, thanks!  

@bparees @adellape  - updated with more official link to the solution

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 3]
```diff
@@ -188,3 +188,24 @@ following:
...
+
+== Additional Configuration and Troubleshooting
+Depending on what version of NFS is being used and how it is configured, there may be additional
+configuration steps needed for proper export and security mapping.  Below are some links and topics that
+may apply:
+
+`*NFSv4 mount incorrectly shows all files with ownership of nobody:nobody*`
+
+- Could be attributed to the ID mapping settings (/etc/idmapd.conf) on your NFS client and server, as NFSv4 utilizes ID mapping to ensure permissions are set properly on exported shares, if the domains of the client and server do not match then the permissions are mapped to nobody:nobody
+- For link:https://access.redhat.com/solutions/33455[more information] on how to diagnose and resolve this issue
```
Comment (bparees) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: hm, i'm guessing linking to content behind a paywall will not be popular in our origin docs.  @adellape ?

Also this sentence is going to read "For _more information_ on how to diagnose and resolve this issue" which is a fragment.

Comment (screeley44) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: @bparees -  see your point on that, is it acceptable to use existing similar solutions/articles from non Red Hat sites - like linuxquestions.com or serverfault.com  OR can we make this solution public on the Red Hat site or ????

Comment (bparees) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: i'm gonna defer to @openshift/team-documentation on that.

**Comment by adellape** [comment]
:+1: Sorry for the delay.

**Comment by adellape** [comment]
**revhistory note:**

Will be added in follow-up PR to come.

**Comment by adellape** [comment]
It looks like this has already been reworked and into the downstream docs: 

https://docs.openshift.com/enterprise/latest/install_config/persistent_storage/persistent_storage_nfs.html#nfs-additional-config-and-troubleshooting

So I'll remove this from the "Future Release" milestone and remove the `to_followup` label.

**Pull request merged**